src/components/studies/scheduler/utility.ts

```diff
@@ -13,3 +14,30 @@ export function getDropdownTimeItems(): StringDictionary<string> {
   return menuItems
 }
 
+export function isSameAsDefaultSchedule(schedule: Schedule) {
+  const defaultSessionDelay = undefined
```

Comment:
it's correct but I think it could be much shorter. e.g.
instead of 
`const defaultSessionDelay = undefined ...
 for (const session of schedule.sessions) {    
  ....const sessionDelay = session.delay ...
  ... 
 const isScheduleDifferentFromDefault =   sessionDelay !== defaultSessionDelay ||`
  just 
  ```  for (const session of schedule.sessions) {    
   const isScheduleDifferentFromDefault = session.delay !== undefined ...```
   
   you can put in the comments that undefined is a default value